tx.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+/*
+message Tx {
+    oneof tx {
+        WitnessTx witness = 1;
+        SignTx    sign    = 2;
+        RelayTx   relay   = 3;
+    }
+}
+
+message RelayTx {
```

Comment:
We don't need a RelayTx as far as I can tell. 

The relayer will just passively read the latest state from the peg zone by listening to events and then send it to Ethereum.